Gemfile

```diff
@@ -90,12 +90,12 @@ gem 'bcrypt', '~> 3.1.6'
 # See: config/initializers/rabl_hack.rb
 gem 'rabl', '~> 0.13.0'
 gem 'multi_json', '~> 1.12.1'
-gem 'oj', '~> 3.0.5'
+gem 'oj', '~> 3.0.6'
```

Comment:
Did look at it but briefly, but from my understanding it focuses mainly on replacing `to_json` on an AR object which we don't do. But we could of course activate it and see if there are any benefits.

But I'd do that separately.